app/src/main/res/layout/activity_chat.xml

```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:fitsSystemWindows="true"
+    android:background="#20000000"
+    tools:context=".ui.ChatActivity">
+
+    <com.google.android.material.appbar.AppBarLayout
+        android:id="@+id/appBar"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <androidx.appcompat.widget.Toolbar
+            android:id="@+id/chatToolBar"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="@color/darkGreen"
+            app:titleTextColor="@color/white"
+            app:menu="@menu/chat_activity_menu">
+
+            <include
+                android:id="@+id/goBack"
+                layout="@layout/back_to_chats_view"/>
+
+            <TextView
+                android:id="@+id/actTitle"
```

Comment:
name casing should always be like tvActTitle (the prefix should tell what type of view is used)